src/any_agent/frameworks/agno.py

```diff
@@ -40,18 +34,19 @@ def _get_model(self, agent_config: AgentConfig):
         )
 
     async def _load_agent(self) -> None:
+        if not agno_available:
+            msg = "You need to `pip install 'any-agent[agno]'` to use this agent"
+            raise ImportError(msg)
+        if self.managed_agents:
+            msg = "Managed agents are not yet supported in Agno agent."
+            raise NotImplementedError(msg)
+
```

Comment:
Why do this here instead of in the init?  I think I shifted it out of _load_agent and into the __init__ last week 😅

Reply:
Right, I forgot to make one update. 

I am removing the __init__ from the subclasses and implementing it in the parent class.

The __init__ with exception that you added is never reached because Python already raises a TypeError when you try to instantiate `AnyAgent()`: `TypeError: Can't instantiate abstract class AnyAgent with abstract methods ...`